### DISCUSSION TITLE
deepin V23 Beta内测版“深度之家”闪退

### DISCUSSION THREAD

**Discussion by Aninana** [linuxdeepin/developer-center#4362] [Troubleshooting | 故障排查] [answered]
在V23 Beta内测版打开“深度之家”时，应用转圈圈，不久就闪退。
![截图_VirtualBox Machine_20230515085300](https://github.com/linuxdeepin/developer-center/assets/130546027/6a5cef32-66bd-4212-8531-f8b338834788)
在终端执行deepin-home，报如下错误：
![1](https://github.com/linuxdeepin/developer-center/assets/130546027/b7b59dab-492b-4c12-a31e-1c0ececb89b4)

**Comment by BLumia** [comment, marked as answer]
应该和 https://github.com/linuxdeepin/developer-center/issues/4348 重复了，这个问题应该已经修复了，可以在前面链接对应的 issue 跟进。
